### PR TITLE
Configure PHPCS

### PR DESCRIPTION
**Pull request by lcobucci**

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    <exclude-pattern>*/tests/Doctrine/Tests/Proxies/__CG__/*</exclude-pattern>
+
+    <rule ref="Doctrine" />
+
+    <rule ref="PSR1.Classes.ClassDeclaration.MultipleClasses">
+        <exclude-pattern>*/tests/*</exclude-pattern>
+    </rule>
+</ruleset>
+
```
Comment (phansys) [inline phpcs.xml.dist]: Please, remove surplus LF.

**Review by phansys — COMMENTED** [review]
Status: Needs work.

File: .travis.yml [hunk 2]
```diff
@@ -78,6 +78,11 @@ jobs:
       before_script: wget https://phpbench.github.io/phpbench/phpbench.phar https://phpbench.github.io/phpbench/phpbench.phar.pubkey
       script: php phpbench.phar run -l dots --report=default
 
+    - stage: Coding standard
+      php: nightly
```
Comment (stof) [inline .travis.yml]: I suggest using a stable PHP version instead. It is less likely to break PHPCS itself.

Comment (lcobucci) [reply .travis.yml]: That will be done on `v3` after we fix all CS violations

**Pull request merged**